Assets/MRTK/Services/InputSystem/FocusProvider.cs

```diff
@@ -512,14 +512,16 @@ public override void Initialize()
                 FindOrCreateUiRaycastCamera();
             }
 
-            var primaryPointerSelectorType = CoreServices.InputSystem?.InputSystemProfile.PointerProfile.PrimaryPointerSelector.Type;
+            var inputSystem = CoreServices.InputSystem;
+
+            var primaryPointerSelectorType = inputSystem.InputSystemProfile.PointerProfile.PrimaryPointerSelector.Type;
```

Comment:
another optimization: the input system profile is cached in the constructor as `inputSystemProfile`

Reply:
Done!